docs/gallery/images/slope.rst

```diff
@@ -0,0 +1,26 @@
+How to make a slope map
+-----------------------
+
+This tutorial show how to make a slope map of the moon from a geographic grid.
+
+Since we are working with geographic coordinates (longitude and latitude), 
+it is crucial to set the correct ellipsoid parameter first (in this case, for the moon). 
+Next, we utilize ``grdgradient`` to calculate the gradient magnitude (i.e., dz/dr). 
+Finally, we convert this gradient to degrees with ``grdmath``.
+
+.. gmtplot::
+
+   # Make a slope map of the moon.
+   gmt set PROJ_ELLIPSOID moon
+   # Calculate gradient magnitude
+   gmt grdgradient @moon_relief_15m -D -Sgradient.nc -fg
```

Comment:
This will leave a trailing gmt.conf file, which is always a bit dangerous. Since it is used only once, better to use --PROJ_ELLIPSOID=moon directly in the grdgradient line

Reply:
Ok, I see. I will fix it (and reorder a bit the text).